mcs/class/Mono.Debugger.Soft/Makefile

```diff
@@ -28,7 +28,7 @@ $(test_output_dir):
 build-dtest: $(test_output_dir)/dtest-app.exe $(test_output_dir)/dtest-excfilter.exe
 
 $(test_output_dir)/dtest-excfilter.exe: Test/dtest-excfilter.il | $(test_output_dir)
-	$(ILASM) -out:$@ /exe /debug Test/dtest-excfilter.il
+	$(ILASM) -out:$@ /exe /debug:embedded Test/dtest-excfilter.il
```

Comment:
Be sure to test both/all, not delete one test to add another.

Reply:
I removed this change, I will do this in another PR.